security/security-design/shared-assets/zerossl-lb-test-certificate-setup/README.md

```diff
@@ -0,0 +1,238 @@
+
+## Prerequisites
+
+ - An OCI tenancy.
+ - An OCI Administrator with appropriate permissions to create and manage load balancers and associated resources.
+ - Created a sample web site deployed on OCI. 
+ 
+ > **Note** For this tutorial, I have created a compute instance called **apache** and installed and configured the Apache HTTPD service on it, running on port 80.
+
+# Create the Load Balancer
+
+Initially, we will create the load balancer and configure the HTTP listener on port 80. This will enable us to obtain the public IP address of the load balancer, which we will need to create the IP address-based TLS certificate. 
+
+> **Note** If you are planning on using a reserved IP address and already know the public IP address, you can skip the creation of the HTTP listener and configure the HTTPS listener from the start.
+
+1. Login to your OCI console and navigate to **Networking**, then **Load Balancer** from the **Load Balancers** menu.
+
+2. Select the appropriate compartment and click `Create Load Balancer`.
+
+3. In the **Add details** screen, provide the following information:
+
+   - Load balancer name: `lb-for-apache`
+
+   Leave the following values as their defaults:
+
+   - Choose visibility type: `Public`
+   - Assign a public IP address: `Ephemeral IP address`
+   - Choose the minimum bandwidth: `10`
+   - Choose the maximum bandwidth: `10`
+   - Choose networking 
+      -> VCN: Choose your VCN
+      -> Subnet: Choose your public subnet
+
+4. Leave all other values as default and click `Next'.
+
+5. On the **Choose backends** screen, click `Add backends`.
+
+6. Select the checkbox next to your compute instance (in my case `apache`) from the list of instances.
+
+![Screenshot for selecting the backend.](lb-select-backend.png "Screenshot for selecting the backend.")
+
+7. Click `Add selected backends`.
+
+8. Leave the port for your backend as port 80.
+
+9. Leave the health check policy as default.
+
+10. Click `Next`.
+
+11. On the **Configure listener** screen, provide the following information:
+
+   - Name: `listener-http`
+   - Specify the type of traffic your listener handles: `HTTP`
+   - Specify the port your listener monitors for ingress traffic: `80`
+
+12. Click `Next`.
+
+13. Since, this is a test, disable **Error logs** and **Access Logs** (if enabled).
+
+![Screenshot for load balancer logging.](lb-logging.png "Screenshot for load balancer logging.")
+
+14. Click `Submit`.
+
+Your load balancer will be created after a few moments.
+
+![Screenshot for load balancer summary.](lb-created.png "Screenshot for load balancer summary.")
+
+> **Note** If you have enabled the network traffic to your HTTPD server, and if it is running, **Backend sets health** shoudl be showing as **OK**. If not, check your networking and security lists.
+
+15. Make a note of the public IP address of your load balancer, e.g., `130.61.69.66`
```

Comment:
No IP numbers!

Reply:
I understand, but this whole tutorial is based on the fact that i'm getting a cert for an IP address, so, to not include an IP address that can be tracked through the tutorial seems strange. The IP is ephemeral and will be dead once I kill the LB (which I will do once the tutorial is published.